src/main/java/com/kamiroo/todomanager/rest/AdministrationRestController.java

```diff
@@ -0,0 +1,29 @@
+package com.kamiroo.todomanager.rest;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AdministrationRestController {
+
+    @Autowired
+    private Environment environment;
+
+    @Value("${app.version}")
+    private String valueFromFile;
```

Comment:
nie powinno się tak nazywać zmiennych, może być wiele wartości z pliku. Zmienne powinny mieć nazwy które jasno określają co one reprezentują. To czy wartość jest z pliku, czy skąd indziej chyba też nie ma znaczenia w tym przypadku. Zmień nazwę zmiennej.